textext.py

```diff
@@ -174,6 +174,50 @@ def __init__(self):
             dest="scale_factor",
             default=self.settings.get('scale', float, 1.0))
 
+    # Identical to inkex.Effect.getDocumentWidth() in Inkscape >= 0.91, but to provide compatibility with
+    # Inkscape 0.48 we implement it here explicitly again as long as we provide compatibility with that version
+    def get_document_width(self):
+        width = self.document.getroot().get('width')
+        if width:
+            return width
+        else:
+            viewbox = self.document.getroot().get('viewBox')
+            if viewbox:
+                return viewbox.split()[2]
+            else:
+                return '0'
```

Comment:
I would go with try-catch approach, instead. You may not account some edge case with `if` statements, so `try-catch` is safer here.

Reply:
In general, I'd agree, but I think, since this is verbatim the same code as in `inkex.py` itself, we can "safely" assume that when it has not yet caused issues for anyone, then it won't for us.

To be safe, go try/catch. Your call.